tests/integration/test_end_point.py

```diff
@@ -195,6 +180,73 @@ def test_health_database_ko(setup_database):
         assert_get_health_200()
 
 
+def assert_health(expected_result):
+    if expected_result == 200:
+        assert_get_health_200()
+    elif expected_result == 503:
+        assert_get_health_with_ko_and_503()
+
+
+def get_health_data_provider():
+    services = list(product([True, False], repeat=4))
+    data_provider = []
+    for service_list in services:
+        data_provider.append(
+            {
+                "expected_result": 200 if all(s is True for s in service_list) else 503,
```

Comment:
If those are boolean you can shorten to
```suggestion
                "expected_result": 200 if all(service_list) else 503,
```